cmd/kubeadm/app/phases/bootstraptoken/clusterinfo/clusterinfo_test.go

```diff
@@ -112,3 +132,74 @@ func TestCreateBootstrapConfigMapIfNotExists(t *testing.T) {
 		}
 	}
 }
+
+func TestCreateClusterInfoRBACRules(t *testing.T) {
+	tests := []struct {
+		name    string
+		client  *clientsetfake.Clientset
+		wantErr bool
+	}{
+		{
+			name:    "the RBAC rules already exist",
+			client:  newMockClientForTest(t),
+			wantErr: false,
+		},
+		{
+			name:    "the RBAC rules do not exist",
+			client:  clientsetfake.NewSimpleClientset(),
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if err := CreateClusterInfoRBACRules(tt.client); (err != nil) != tt.wantErr {
```

Comment:
if you remove `wantErr` from `tests`, then you can just update this piece of code as below, 

```golang
if err := CreateClusterInfoRBACRules(tt.client); err!=nil {
    t.Errorf(...)
}